ci-operator/populate-owners.sh

```diff
@@ -1,42 +0,0 @@
-#!/bin/bash
```

Comment:
One last comment -- can you make this script run your tool with the correct inputs and outputs? This keeps a consistent API for repo admins

Reply:
> ... can you make this script run your tool with the correct inputs and outputs?

It doesn't really have inputs and outputs, you just run it with your `$PWD` somewhere inside the repository (the path comes in [here][1] which gets [fed to `getRepoRoot`][2] ).  But I've added a wrapper with 8bcba67 -> 9fbc825, which also:

* Updates `OWNERS` and `OWNERS_ALIASES` with a fresh run.
* Adds `os.O_TRUNC` to avoid occasional corruption like:

    ```diff
    diff --git a/ci-operator/jobs/openshift/azure-misc/OWNERS b/ci-operator/jobs/openshift/azure-misc/OWNERS
    index 132c684..bf0dec5 100644
    --- a/ci-operator/jobs/openshift/azure-misc/OWNERS
    +++ b/ci-operator/jobs/openshift/azure-misc/OWNERS
    @@ -3,10 +3,11 @@
     # See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
  
     approvers:
    +- pweil-
     - jim-minter
     - kargakis
     - kwoodson
    -- mjudeikis
    +is
     - pweil-
     reviewers:
     - jim-minter
    ```
    
    although the exact source of that corruption is not clear to me.



[1]: https://github.com/openshift/release/pull/1285/commits/e1f993fb1debc43d81f7c413cd4fea39077d48b4#diff-71d56af6d838dcb321a05bee9c81bda1R327
[2]: https://github.com/openshift/release/pull/1285/commits/e1f993fb1debc43d81f7c413cd4fea39077d48b4#diff-71d56af6d838dcb321a05bee9c81bda1R289